test/stdlib/test_helper.rb

```diff
@@ -411,15 +106,33 @@ def each(&block)
   end
 end
 
-class ArefFromStringToString < BlankSlate
+class ArefFromStringToString < RBS::UnitTest::Convertibles::BlankSlate
   def [](str)
     "!"
   end
 end
 
+BlankSlate = RBS::UnitTest::Convertibles::BlankSlate
+ToIO = RBS::UnitTest::Convertibles::ToIO
+ToI = RBS::UnitTest::Convertibles::ToI
+ToInt = RBS::UnitTest::Convertibles::ToInt
+ToF = RBS::UnitTest::Convertibles::ToF
+ToR = RBS::UnitTest::Convertibles::ToR
+ToC = RBS::UnitTest::Convertibles::ToC
+ToStr = RBS::UnitTest::Convertibles::ToStr
+ToS = RBS::UnitTest::Convertibles::ToS
+ToSym = RBS::UnitTest::Convertibles::ToSym
+ToA = RBS::UnitTest::Convertibles::ToA
+ToArray = RBS::UnitTest::Convertibles::ToArray
+ToHash = RBS::UnitTest::Convertibles::ToHash
+ToPath = RBS::UnitTest::Convertibles::ToPath
+CustomRange = RBS::UnitTest::Convertibles::CustomRange
+Each = RBS::UnitTest::Convertibles::Each
```

Comment:
`include RBS::UnitTest::Convertibles`?

Reply:
Ah this is in fact for RBS runtime type checker. I'm not sure if including works for it...